tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs

```diff
@@ -56,22 +60,83 @@ private static string resolveRepoLocation(string storageLocation = null)
         /// <param name="args">CommandLineParser arguments. In server mode use double dash '--' and everything after that becomes additional arguments to Host.CreateDefaultBuilder. Ex. -- arg1 value1 arg2 value2 </param>
         public static async Task Main(string[] args = null)
         {
-            VerifyVerb(args);
+            var verb = VerifyVerb(args);
             var parser = new Parser(settings =>
             {
                 settings.CaseSensitive = false;
                 settings.HelpWriter = System.Console.Out;
                 settings.EnableDashDash = true;
             });
 
-            await parser.ParseArguments<StartOptions, PushOptions, ResetOptions, RestoreOptions>(args)
-                .WithNotParsed(ExitWithError)
-                .WithParsedAsync(Run);
+            if (verb == "config")
+            {
+                StringBuilder helpTextBuilder = new StringBuilder();
+                StringWriter helpTextWriter = new StringWriter(helpTextBuilder);
+
+                var subParser = new Parser(settings =>
+                {
+                    settings.CaseSensitive = false;
+                    settings.HelpWriter = helpTextWriter;
+                    settings.EnableDashDash = true;
+                });
+
+                var parseResult = subParser.ParseArguments<ShowOptions, LocateOptions>(args.Skip(1));
+                parseResult.WithNotParsed(errs => SubVerbExitWithError(parseResult, errs, helpTextBuilder.ToString()));
+                await parseResult.WithParsedAsync(Run);
+            }
+            else
+            {
+                await parser.ParseArguments<StartOptions, PushOptions, ResetOptions, RestoreOptions, ConfigOptions>(args)
+                    .WithNotParsed(ExitWithError)
+                    .WithParsedAsync(Run);
+            }
         }
 
-        static void ExitWithError(IEnumerable<Error> errors)
+        static void SubVerbExitWithError<T>(ParserResult<T> result, IEnumerable<Error> errors, string defaultHelpText)
```

Comment:
Can this be encapsulated in ConfigOptions class as all the subverbs related to "config"?

Reply:
Maybe. I'll try.